definitions/yok.d.ts

```diff
@@ -24,6 +26,8 @@ interface IInjector extends IDisposable {
 	isValidHierarchicalCommand(commandName: string, commandArguments: string[]): boolean;
 	getChildrenCommandsNames(commandName: string): string[];
 	buildHierarchicalCommand(parentCommandName: string, commandLineArguments: string[]): any;
+	publicApi: any;
+	_publicApi: any;
```

Comment:
sounds good and I think it will work :) 
I'll change it, thanks for the idea :)